src/browser_app_skeleton/webpack.mix.js

```diff
@@ -9,6 +9,13 @@
 
 let mix = require("laravel-mix");
 
+// Customize the notifier to be less noisy
+let WebpackNotifierPlugin = require('webpack-notifier');
+let webpackNotifier = new WebpackNotifierPlugin({
+  alwaysNotify: false,
+  skipFirstNotification: true
+})
```

Comment:
This also makes it so that Laravel Mix logo/title is not displayed. This will be a lot less confusing to new users

Reply:
#249